examples/tf/cluster_demo.py

```diff
@@ -21,30 +22,34 @@ def run_task(v):
     run_experiment can easily execute variants of the experiment on different
     machines
     """
-    env = TfEnv(env_name="CartPole-v1")
+    with LocalRunner() as runner:
```

Comment:
hmm now that i look at the code, maybe these should be called "trainers". i wonder, are they algorithm trainers or experiment runners

Reply:
Yes, right now it is more appropriate to call it "trainer". My idea is that ultimately we remove run_experiment, and replace it with `LocalRunner`, `RemoteRunner`, etc. The Runner take care of the logic of both setting up an experiment and training it.

Reply:
sgtm